package-overrides/github/components/jquery@2.0.3.json

```diff
@@ -1,5 +0,0 @@
-{
-  "main": "jquery",
-  "files": ["jquery.js"],
-  "format": "global"
-}
```

Comment:
I don't think we so. They are harmless as there is no need to maintain them. I would say there is no gain for breaking someone's code in this case, as unlikely as that may be.